sentry-sample/build.gradle.kts

```diff
@@ -9,14 +9,21 @@ android {
 
     defaultConfig {
         applicationId = "io.sentry.sample"
-        minSdkVersion(Config.Android.minSdkVersion)
+        minSdkVersion(Config.Android.minSdkVersionNdk)
         targetSdkVersion(Config.Android.targetSdkVersion)
         versionCode = 1
         versionName = "1.0"
 
         externalNativeBuild {
+            val sentryNativeSrc = if (File("${project.projectDir}/sentry-native-local").exists()) {
```

Comment:
actually it needs to check `../sentry-android-ndk/sentry-native-local` will fix it tomorrow (or today?) :D